zospy/analyses/base.py

```diff
@@ -324,9 +324,8 @@ def Results(self) -> _ZOSAPI.Analysis.Data.IAR_:  # noqa: N802
         """Analysis results."""
         return self._analysis.GetResults()
 
-    @property
-    def field(self) -> int | str:
-        """Gets the wavelength value of the analysis.
+    def get_field(self) -> int | str:
+        """Get the field value of the analysis.
```

Comment:
```suggestion
        """Get the field value from the analysis settings.
```
The docstrings can state more explicitly that these methods access / modify the analysis settings.